script/testing/junit/src/SelectTest.java

```diff
@@ -130,11 +130,9 @@ public void testSelectAlias() throws SQLException {
         assertNoMoreRows(rs);
 
         /* Wrong count, wait for Having fix */
-        select_SQL = "SELECT COUNT(*) AS cnt FROM tbl HAVING COUNT() > 10;";
+        select_SQL = "SELECT COUNT(*) AS cnt FROM tbl HAVING COUNT(*) > 10;";
         stmt = conn.createStatement();
         rs = stmt.executeQuery(select_SQL);
```

Comment:
Keep checkRow function call to verify the result, but change it to 0

Reply:
Query: SELECT COUNT(*) AS cnt FROM tbl HAVING COUNT(*) > 10;
In this query, cnt value will always be 2. Since we have a check of having count(*) > 10, no rows will be returned. I'm validating this with help of assertNoMoreRows. I don't think we can use checkRow function in this case.
Please let me know if I'm wrong.

Reply:
Ok then it should be good to go.